src/bls_ops.rs

```diff
@@ -305,19 +308,19 @@ pub fn op_bls_verify(a: &mut Allocator, input: NodePtr, max_cost: Cost) -> Respo
     // followed by a variable number of (G1, msg)-pairs (as a flat list)
     args = rest(a, args)?;
 
-    let mut items = Vec::<(PublicKey, &[u8])>::new();
+    let mut items = Vec::<(PublicKey, Vec<u8>)>::new();
```

Comment:
I'm not really excited about this copy.  I don't believe we have any constraints on the size of this atom, so you can OOM the interpreter here, potentially.
But this is a case where the current implementation actually is broken.
I believe `aggregate_verify` works with anything implements `Borrow<[u8]>`. Would it be feasible to have `Atom` implement that?